src/Transition/CreateNewIDFUsingRulesV23_2_0.f90

```diff
@@ -308,6 +308,7 @@ SUBROUTINE CreateNewIDFUsingRules(EndOfFile,DiffOnly,InLfn,AskForInput,InputFile
           NumUnitarySystem = GetNumObjectsFound('AIRLOOPHVAC:UNITARYSYSTEM')
           NumUnitaryWAHP = GetNumObjectsFound('AIRLOOPHVAC:UNITARYHEATPUMP:WATERTOAIR')
           NumZoneWAHP = GetNumObjectsFound('ZONEHVAC:WATERTOAIRHEATPUMP')
+          IF (ALLOCATED(CoilLatentStuff)) DEALLOCATE(CoilLatentStuff)
```

Comment:
@amirroth encountered an issue during a multi-file transition where this was already allocated, so the transition failed.  I actually couldn't reproduce this locally...perhaps GCC is cool with just allocating again?  But either way, this change did not break anything for me, and should fix the issue there.  Worth a final test of course.